skycoin/dmsg
# Fix wrong current client and server connected

Fixes #119 

 Changes:	
- set `delEntry` on close servers
- remove *pubkey* on `delEntry` from **servers** OR **clients** set store in Redis db.
- set expire timeout on server type entries, and add routine for  delete old entries on `servers` set record.

How to test this PR:
- run dmsg-discover and dmsg-server (suggest to run 2 server) as test (following [these](https://github.com/skycoin/dmsg/tree/develop/integration) steps).
- on develop@skywire, generate a config and replace dmsg discovery address with `http://localhost:9090`
- run the visor
- check redis for keys
- close visor or server and check redis again.
